webhook-pubsub-publisher/src/app.ts

```diff
@@ -19,7 +19,7 @@ import type {Probot} from 'probot';
 
 export default (app: Probot): void => {
   app.onAny(async event => {
-    app.log(`Received "${event.name}" event with ID ${event.id}`);
+    app.log(event, 'Received event');
```

Comment:
```suggestion
    app.log('Received event', event);
```

(although I'm not sure if `app.log` takes more than one arg?)

Reply:
Yeah I checked, probot uses `pino` so the way I had it was correct

https://getpino.io/#/docs/api?id=loggerinfomergingobject-message-interpolationvalues

Reply:
Looks like it should be something like this then:
```suggestion
    app.log('Received event: %O', event);
```

* https://getpino.io/#/docs/api?id=interpolationvalues-any
* https://nodejs.org/api/util.html#utilformatformat-args